mflowgen/glb_top/custom-init/outputs/floorplan.tcl

```diff
@@ -52,4 +52,22 @@ setFlipping s
 
 planDesign
 
+set tiles [get_cells *glb_tile*]
+set tile_width [dbGet [dbGet -p top.insts.name *glb_tile* -i 0].cell.size_x]
+set tile_start_y 20
+set tile_start_x 60
```

Comment:
Same as above. It'd be better to automatically center the tiles within the glb_top floorplan.